pkgs/applications/virtualization/rkt/default.nix

```diff
@@ -2,11 +2,11 @@
 , fetchurl, fetchFromGitHub }:
 
 let
-  coreosImageRelease = "835.9.0";
-  coreosImageSystemdVersion = "225";
+  coreosImageRelease = "794.1.0";
```

Comment:
Just using the same version of the coreos image to build the stage1 as rkt's build scripts use by default. In my last PR I bumped it to the latest stable without knowing that there was a specific version that rkt uses.